source/_patterns/03-organisms/global/header.twig

```diff
@@ -9,5 +9,45 @@
       }
     %}
   </div>
-  {% include '@organisms/global/main-navigation.twig' %}
+  <div class="c-main-header__inner js-stop-prop">
+
+    <button class="c-menu-toggle js-toggle" data-toggled="c-body" data-prefix="side-menu" id="menu-toggle" aria-expanded="false">
+      <span class="is-vishidden">Menu</span>
+      <span class="c-menu-toggle__line"></span>
+    </button>
+
+    {% include '@organisms/global/side-menu.twig' %}
+
+    <div class="c-main-header__branding">
+      {% include '@atoms/images/gothamist-logo.twig' %}
+    </div>
+
+    {%
+      include '@molecules/navigation/primary.twig'
+      with {
+        "primary_nav_classes": "c-primary-nav--top"
+      }
+    %}
+
+    <div class="c-main-header__right">
+      {%
+        include '@atoms/buttons/button.twig'
+        with {
+          "button_text": "Donate",
+          "button_classes": "c-main-header__donate o-button--sm"
+        }
+      %}
+      <button class="c-search-toggle js-toggle" data-prefix="top-search" data-toggled="c-main-header__right" id="search-toggle" aria-expanded="false">
```

Comment:
shouldn't this button be a "quadrinany" button, rather than something specific to the header?

Reply:
Maybe? The styles of it were unique due to it being just a button with an icon in it and no text. If an instance of this usage pops up again, I'll create an object style for it.